nodeshot/ui/default/templates/index.html

```diff
@@ -29,7 +29,7 @@
 
     <script id="search-template" type="text/template">
         <input type="text" id="general-search-input" placeholder="{% trans 'Search for data' %}">
-        <span class="icon-search">&nbsp;</span>
+        <span class="icon-search cursor-pointer" id="general-search-input-button">&nbsp;</span>
```

Comment:
if the previous suggestions work, `id="general-search-input-button"`can be removed